src/redux/reducers/index.js

```diff
@@ -46,6 +50,22 @@ function nativeLanguage(state = '', action) {
       return state;
   }
 }
+
+// Seen phrases reducer
+function seenPhrases(state = [], action) {
+  switch (action.type) {
+    case SET_SEEN_PHRASE:
+      return action.payload;
+    // We need the seen phrases key type
+    case SEEN_PHRASES_KEY:
+      return action.payload;
+    // Removes phrases from the seen phrases  
+    case REMOVE_FROM_SEEN_PHRASES:
+      return action.payload;
```

Comment:
delete REMOVE_FROM_SEEN_PHRASES and SEEN_PHRASES_KEY they can be just replaced by SET_SEEN_PHRASE

Reply:
delete REMOVE_FROM_SEEN_PHRASES